src/pages/Collections/CollectionPage.ViewModel.ts

```diff
@@ -330,4 +429,1711 @@ export class CollectionPageViewModel {
       }
     }
   };
+
+  /**
+   * Handle updating tab
+   * @param data :any - tab data i.e. collection, folder or request
+   * @param route :string - path to collection, folder or request
+   * @param _id :string - if of the tab
+   */
```

Comment:
type not required in js-doc comment as we are already using typescript, same at multiple places